aidial_adapter_vertexai/chat/gemini/adapter.py

```diff
@@ -60,6 +60,8 @@
     HarmCategory.HARM_CATEGORY_HARASSMENT: HarmBlockThreshold.BLOCK_ONLY_HIGH,
 }
 
+DEFAULT_CITATIONS_LIMIT = 10
```

Comment:
I don't think we need to hind anything from the user. Whatever the model returned, the user have the right get its full unabridged content.

Reply:
I was just sacred of 40 links that poped up on Harry Potter example :)

Fixed